### ISSUE TITLE
{Linked:Bug13061835}[Adaptive Card- Designer] - My Expenses link under card is not accessible through the talkback swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#276]
### Target Platforms

Android

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: Samsung S20
Android Version: 13
Application: Adaptive Cards Visualizer 
Screen Reader: Talkback

**Pre-Requisite:**
Upload the Json from device.

**Repro Steps:**
1. Open the Adaptive Card application.
2. Navigate to the 'Import' tab using right swipe gesture and double tap on it to select it.
3. Perform right swipe and navigate to the Choose file button and double tap to activate it.
4. Select any file.
5. Now perform right swipe and navigate to the 'Visual' tab and double tap on it to activate it.
6. Navigate the card using swipe gesture and observe the issue.

**Actual Result:**
My Expenses link present on the card is not accessible through the swipe gesture. While navigating on the card using right swipe gesture focus is landing on the entire label but it is not landing on the my expenses link.

**Expected Result:**
My Expenses link should be accessible through the swipe gesture and user perform left or right swipe then screen reader focus should be land on the link.

**Note:**
Same issue is repro while navigating using keyboard.

**User Impact:**
Users who rely on screen readers will be impacted When talkback is enabled, "My expense" link present in the card is not accessible as they will not be  able to take action on the link.

**WCAG Reference:**
[https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/348e448b-799c-45d4-8bd8-c07140617ed6

[MyExpenseLink_adaptivecard_payload.json](https://github.com/user-attachments/files/18375045/MyExpenseLink_adaptivecard_payload.json)

### Card JSON

```JSON
na
```

**Comment by vagpt** [comment]
Hi @GabrielMedAlv,

Hope you are doing well.

Is there any update on this bug?

**Comment by vagpt** [comment]
Hi @GabrielMedAlv,

Gentle Reminder...

**Comment by ardlank** [comment]
@vagpt seems like this issue only comes on the emulator. I tested this out on physical device and I am able to double tap and go to the link. Please test on a physical device

CC: @VikrantSingh01

**Comment by vagpt** [comment]
Hi @ardlank,

Thanks for the updates.

This issue we have tested on Physical device only using the attached JSON so could you please check it again?

Or also, could you please provide us component name to direct verify this at our end?
